examples/scala/src/main/scala/example/Streaming.scala

```diff
@@ -45,14 +46,17 @@ object Streaming {
     val df = spark.read.format("delta").load(path)
     df.show()
 
+
+    println("=== Section 2: write and read delta stream")
     println("Streaming write")
```

Comment:
stream -> "using structured streaming", and we can get rid of the old println that was there before.

Reply:
Yes, I updated with your comment.